app/assets/stylesheets/mortgage_calculator/components/_affordability.scss

```diff
@@ -54,10 +55,34 @@
     @include respond-to($mq-m) {
       @include column(6);
       @include push(1);
+      position: absolute;
```

Comment:
This is a bit problematic at a certain viewport, doesn't seem like it should need to be absolutely positioned?

![image](https://cloud.githubusercontent.com/assets/14920201/15502679/778decfc-21ad-11e6-895c-de10a9b3f397.png)